OrbitQt/main.cpp

```diff
@@ -253,8 +253,14 @@ int main(int argc, char* argv[]) {
 #endif
 
     QApplication app(argc, argv);
-    QCoreApplication::setApplicationName("Orbit Profiler [BETA]");
-    QCoreApplication::setApplicationVersion(QString::fromStdString(OrbitCore::GetVersion()));
+    QApplication::setApplicationName("orbitprofiler");
+
+    // The application display name is automatically appended to all window titles when shown in the
+    // title bar: <specific window title> - <application display name>
+    const auto versionString = QString::fromStdString(OrbitCore::GetVersion());
```

Comment:
`version_string`? Or are we sticking with Qt's casing here?

Reply:
Ahh. Yesss.. Sorry. my bad. I will fix it.